src/Bridge/Graphql/Resolver/ItemResolverFactory.php

```diff
@@ -81,7 +84,13 @@ public function createItemResolver(string $resourceClass, string $rootClass): ca
                 $item = $this->itemDataProvider->getItem($resourceClass, \count($identifiers) > 1 ? \implode(';', $identifiers) : $uniqueIdentifier[0]);
             }
 
-            return $item ? $this->normalizer->normalize($item, null, ['graphql' => true]) : null;
+            return $item ? $this->normalizer->normalize(
+                $item,
+                null,
+                ['graphql' => true] + $this->resourceMetadataFactory
+                    ->create($resourceClass)
```

Comment:
Can't this be `null` if metadata hasn't been found?

Reply:
Why? Should not the `ResourceClassNotFoundException` be thrown instead?

Reply:
Monday :sleeping: